rest_framework/metadata.py

```diff
@@ -107,10 +114,9 @@ def get_serializer_info(self, serializer):
             # If this is a `ListSerializer` then we want to examine the
             # underlying child serializer instance instead.
             serializer = serializer.child
-        return OrderedDict([
-            (field_name, self.get_field_info(field))
-            for field_name, field in serializer.fields.items()
-        ])
+        return OrderedDict([(field_name, self.get_field_info(field))
+                            for field_name, field in
+                            serializer.fields.items()])
```

Comment:
Is this different indentation style mandatory? It reads less obviously to me. If we can avoid changing it let's do so.